src/test/java/com/epam/service/PersonDetailsServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.service;
+
+import com.epam.model.Person;
+import com.epam.model.PersonRoleEnum;
+import com.epam.repository.PersonDAO;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.when;
+import static org.junit.jupiter.api.Assertions.*;
+
+class PersonDetailsServiceImplTest {
```

Comment:
Hmm, I pretty sure this class should be named smth like this: `PersonServiceTest`, what do u think?

Reply:
Yeup, u re right. Fixed